build-system/tasks/runtime-test/helpers.js

```diff
@@ -15,21 +15,37 @@
  */
 'use strict';
 
-const colors = require('ansi-colors');
+const argv = require('minimist')(process.argv.slice(2));
 const deglob = require('globs-to-files');
 const findImports = require('find-imports');
 const fs = require('fs');
+const gulp = require('gulp');
 const log = require('fancy-log');
 const minimatch = require('minimatch');
+const opn = require('opn');
 const path = require('path');
+const webserver = require('gulp-webserver');
+
+const {
+  reportTestErrored,
+  reportTestFinished,
+  reportTestSkipped,
+  reportTestStarted,
+} = require('../report-test-status');
+const {app} = require('../../test-server');
 const {exec} = require('../../exec');
 const {gitDiffNameOnlyMaster} = require('../../git');
-const {green, cyan, red} = colors;
+const {green, yellow, cyan, red} = require('ansi-colors');
 const {isTravisBuild} = require('../../travis');
-const extensionsCssMapPath = 'EXTENSIONS_CSS_MAP';
+const {Server} = require('karma');
 
-const ROOT_DIR = path.resolve(__dirname, '../../../');
+const BATCHSIZE = 4; // Number of Sauce Lab browsers
+const EXTENSIONSCSSMAP = 'EXTENSIONS_CSS_MAP';
```

Comment:
Remember to change this comment in `css.js`

https://github.com/ampproject/amphtml/blob/83cfcfff825012e7b2dc562f0841bb0c0486527b/build-system/tasks/css.js#L115-L116